README.md

```diff
@@ -5,6 +5,21 @@
 Holds generated but persistent results from building the `ansible` community package.  This information
 may be referred to by other projects and scripts.
 
+## Issue tracker
+
+The [issue track of this repository](https://github.com/ansible-community/ansible-build-data/issues) is to track various aspects of the `ansible` build, including:
```

Comment:
```suggestion
This repository's [issue tracker](https://github.com/ansible-community/ansible-build-data/issues) handles various aspects of the `ansible` build, including:
```

Reply:
I would modify it slightly to include `This repository's` in the link title. This makes it easier to understand what the link is about for screen readers. Do you want to update your suggestion?

Reply:
Done!